src/modules/blocks/create-note.js

```diff
@@ -0,0 +1,36 @@
+import * as WebImporter from '@adobe/helix-importer';
+
+// For list of icons refer - https://github.com/adobe-experience-league/exlm/tree/main/icons
+const iconMapping = {
+  important: 'alert',
+  warning: 'warning',
+  caution: 'warning',
+  error: 'warning',
+};
+
+export default function createNote(document) {
+  // Notes have class extension along with the variation name.
+  const noteElements = Array.from(
+    document.querySelectorAll('.extension:not(.relatedarticles):not(.video)'),
```

Comment:
The HTML for note has a `note` class, why not use that?

Reply:
`note` is just one of the variations, there are other variations too. Excluding `video` and `relatedarticles` which are captured as different blocks, everything else falls under the [note block](https://jira.corp.adobe.com/browse/EXLM-89).